examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -1,47 +1,44 @@
 # Fit to multiple relative binding free energy edges
+import datetime
 import sys
 from argparse import ArgumentParser
+from collections import defaultdict, namedtuple
+from pathlib import Path
+from pickle import dump, load
+from time import time
+from typing import Any, Dict, List, Tuple, Union
+
 import jax
-from jax import numpy as jnp
 import numpy as np
-import datetime
-import timemachine
-from timemachine.training.dataset import Dataset
+from jax import numpy as jnp
 
-# forcefield handlers
-from timemachine.ff import Forcefield
-from timemachine.ff.handlers.serialize import serialize_handlers
-from timemachine.ff.handlers.deserialize import deserialize_handlers
-from timemachine.ff.handlers.nonbonded import AM1CCCHandler, LennardJonesHandler
+import timemachine
+from timemachine.fe.estimator import SimulationResult
 
 # free energy classes
 from timemachine.fe.free_energy import (
+    RBFETransformIndex,
     RelativeFreeEnergy,
     construct_lambda_schedule,
-    RBFETransformIndex,
 )
-from timemachine.fe.estimator import SimulationResult
-from timemachine.fe.model import RBFEModel
 from timemachine.fe.loss import pseudo_huber_loss  # , l1_loss, flat_bottom_loss
+from timemachine.fe.model import RBFEModel
+
+# forcefield handlers
+from timemachine.ff import Forcefield
+from timemachine.ff.handlers.deserialize import deserialize_handlers
+from timemachine.ff.handlers.nonbonded import AM1CCCHandler, LennardJonesHandler
+from timemachine.ff.handlers.serialize import serialize_handlers
 
 # MD initialization
 from timemachine.md import builders
+from timemachine.optimize.step import truncated_step
```

Comment:
unfortunately `isort` does seem to mangle annotated import groups. Interleaved comments keep the groups separate, so we might just manually fix these cases.